src/test/resources/simple-using-variable.ktr

```diff
@@ -0,0 +1,792 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<transformation>
+  <info>
+    <name>simple-using-variable</name>
+    <description/>
+    <extended_description/>
+    <trans_version/>
+    <trans_type>Normal</trans_type>
+    <trans_status>0</trans_status>
+    <directory>/</directory>
+    <parameters>
+      <parameter>
+        <name>title</name>
+        <default_value>Prof.</default_value>
+        <description/>
+      </parameter>
+    </parameters>
```

Comment:
We shouldn't be setting `title` as a parameter - only as a variable.

Reply:
Done